FordLabs/retroquest
# Datemigration2

## Overview
This will drop the existing date_created and last_login_date fields in the Board, Team, and Feedback tables. It will then rename the newly added date_created2 and last_login_date2 fields to the original names of date_created and last_login_date. It will also delete the dateMigrationConfiguration created in the last PR.

### Notes
Please verify data in database correctly has the date_created2 and last_login_date2 fields filled out before merging this branch.

## Testing Instructions
How to test this PR

- start from a clean MySQL db on the develop branch with the datemigration and dockerdb profiles
- create a board in the ui
- add a few thoughts
- click end retro
- add a Feedback
- log out
- log back in
- change to the datemigration2 branch
- startup the app
- check that there are no longer date_created2 or last_login_date2 fields in the MySQL database (Team, Board, Feedback) and that their types are one of DATE or TIMESTAMP and not TINYBLOB

Everything should look good, and the ui should still function the same.
